src/js/components/visualization/ChartComponent.jsx

```diff
@@ -114,7 +114,8 @@ export class ChartComponent extends Component {
 
           this._chart.drilldown({
             ...chartData,
-            data: (seriesItem || dataItem)[drillDownFieldName]
```

Comment:
I don't see the difference here? either way, if `seriesItem[drillDownFieldName]` is undefined, than its going to end up being `dataItem[drillDownFieldName]` ?